locales/en.json

```diff
@@ -0,0 +1,34 @@
+{
+	"greeting": "Here you can attest your email address.\n\nYour email will be linked to your Byteball address, the link can be either made public (if you choose so) or saved privately in your wallet. In the latter case, only a proof of attestation will be posted publicly on the distributed ledger. \n\nThe price of attestation is {{priceInBytes}} GB. The payment is nonrefundable even if the attestation fails for any reason.\n\nAfter payment, you will be able to prove your email by receiving a verification code and entering it here.",
```

Comment:
{{priceInBytes}} but GB?

Reply:
Fixed this and multiple other places where placeholder name didn't match with the data it contains.